editor.planx.uk/src/components/Header/Header.tsx

```diff
@@ -231,58 +235,70 @@ const Breadcrumbs: React.FC = () => {
     state.previewEnvironment === "standalone",
   ]);
 
+  const flowStatus = useStore.getState().status as StatusVariant | undefined;
+
+  console.log(flowStatus);
```

Comment:
```suggestion
  const flowStatus = useStore(state => state.flowStatus);
```

Within React components we should be using the hook variant of our Zustand store (`useStore()` as a function). We only really need to reach for the non-hook version outside React components - helper functions or tests generally.

Reply:
Thanks for this, very useful to know, I had copied the `.getState()` function from elsewhere, have updated